proto/perception.proto

```
@@ -52,6 +52,30 @@ enum GestureType {
   GESTURE_TYPE_CANCEL      = 4;
 }
 
+// HandGestureType classifies the physical hand pose detected by the gesture
+// classifier. Distinct from GestureType, which encodes semantic intent.
+// This enum is the canonical type for physical single-hand poses.
+enum HandGestureType {
+  HAND_GESTURE_UNSPECIFIED = 0;
```

Buf (1.67.0)

[error] Enum value name "HAND_GESTURE_UNSPECIFIED" should be prefixed with "HAND_GESTURE_TYPE_".

(ENUM_VALUE_PREFIX)